src/components/Navbar.tsx

```diff
@@ -1,44 +1,138 @@
-          aria-label="Cart"
-        >
-          <div >
-            
-            <svg xmlns="http://www.w3.org/2000/svg" width="26px" height="26px" fill="currentColor" viewBox="0 0 256 256">
-              <path d="M222.14,58.87A8,8,0,0,0,216,56H54.68L49.79,29.14A16,16,0,0,0,34.05,16H16a8,8,0,0,0,0,16h18L59.56,172.29a24,24,0,0,0,5.33,11.27,28,28,0,1,0,44.4,8.44h45.42A27.75,27.75,0,0,0,152,204a28,28,0,1,0,28-28H83.17a8,8,0,0,1-7.87-6.57L72.13,152h116a24,24,0,0,0,23.61-19.71l12.16-66.86A8,8,0,0,0,222.14,58.87ZM96,204a12,12,0,1,1-12-12A12,12,0,0,1,96,204Zm96,0a12,12,0,1,1-12-12A12,12,0,0,1,192,204Zm4-74.57A8,8,0,0,1,188.1,136H69.22L57.59,72H206.41Z" />
-            </svg>
-          </div>
-        </Link>
-        <Link href="/admin/products/manage" className="flex max-w-[480px] cursor-pointer items-center justify-center overflow-hidden rounded-xl h-12 text-[#171212] gap-2 text-xl font-extrabold leading-normal tracking-[0.015em] min-w-0 ">
-          <div>
-          <svg
-      xmlns="http://www.w3.org/2000/svg"
-      width="26px"
-      height="26px"
-      fill="currentColor"
-      viewBox="0 0 256 256"
-      
-    >
-      <path d="M128,24A104,104,0,1,0,232,128,104.11791,104.11791,0,0,0,128,24Zm0,32a40,40,0,1,1-40,40A40.00039,40.00039,0,0,1,128,56Zm0,152a87.69613,87.69613,0,0,1-64-28.93066c.31152-21.30762,42.667-33.06934,64-33.06934s63.68848,11.76172,64,33.06934A87.69613,87.69613,0,0,1,128,208Z"/>
+    <header className="bg-blue-50 px-6 py-4 border-b border-neutral-200 shadow-sm">
+      <div className="max-w-7xl mx-auto flex flex-col md:flex-row items-center justify-between gap-4">
+        
+        <div className="flex items-center justify-between w-full md:w-auto">
+          <Link href="/" className="text-2xl font-bold text-[#171212] hover:text-black font-sans tracking-wide">
+            Home
+          </Link>
+
+          
+          <button onClick={() => setMenuOpen(!menuOpen)} className="md:hidden p-2">
+            {menuOpen ? <X size={26} /> : <Menu size={26} />}
+          </button>
+        </div>
+
+        
+        <nav className="hidden md:flex flex-wrap justify-center gap-6 text-lg font-semibold font-sans text-neutral-700">
+          {categories.map((category) => (
+            <Link
+              key={category}
+              href={`/products?category=${encodeURIComponent(category)}`}
+              className="hover:text-neutral-900 transition-colors"
+            >
+              {category}
+            </Link>
+          ))}
+        </nav>
+
+        
+        <div className="flex items-center justify-center md:justify-end w-full md:w-auto gap-4 relative">
+          
+          <Link
+  href="/cart"
+  className="relative flex items-center justify-center rounded-full p-2 hover:bg-neutral-200 transition"
+  aria-label="Cart"
+>
+  <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="#171212" viewBox="0 0 16 16">
```

Comment:
SVG images added directly in the code are hard to comprehend. Try to create a SVG file with a descriptive name for every SVG and add them as Image element instead